src/apps/api/views/competitions.py

```diff
@@ -141,7 +141,17 @@ def update(self, request, *args, **kwargs):
         in the response to remove a GET from the frontend"""
         partial = kwargs.pop('partial', False)
         instance = self.get_object()
-        serializer = self.get_serializer(instance, data=request.data, partial=partial)
+        data = request.data
+
+        # save leaderboard indiviually, then pass pk to each phase
+        if 'leaderboards' in data:
```

Comment:
Let's leave a `TODO` comment here as well, just so this piece of code is easy to find, and easy to interpret. The comment should mention that 

> this is a temporary fix, so that leaderboards can be saved from the competition edit page. Submission rules and multiple leaderboards still need to be implemented.

Reply:
Oops. Merged this when I saw you approved it. I'll add it to my other PR #542 so it will be there when we merge that.